src/caw/client/cpp_client/caw_client.cpp

```diff
@@ -196,6 +196,44 @@ bool CawClient::GetProfile(const std::string &username) {
   return profileSuccessBool;
 }
 
+bool CawClient::StreamHashtag(const std::string &hashtag) {
+  std::cout << "Streaming Caws with hashtag: #" << hashtag << std::endl;
+  HashtagRequest hashtag_request;
+  hashtag_request.set_hashtag(hashtag);
+  Any request;
+  request.PackFrom(hashtag_request);
+  // Reads and prints the Caw
+  std::function<void(EventReply)> callback = [](EventReply event_reply) {
```

Comment:
Maybe add a more descriptive name for this callback? You could go with FazClientCallback

Reply:
Yeah, I like that